client/src/components/GameBoard/GameBoard.tsx

```diff
@@ -168,17 +169,13 @@ const GameBoard: React.FC<GameBoardProps> = ({ pack }) => {
       startRound(currentPlayerRef.current);
     } else if (gameStatus === "ROUND_READY") {
       nextPlayer();
-    } else if (gameStatus === "FINISHED") {
-      resetGame();
     }
   };
   const buttonText =
     gameStatus === "READY"
       ? "Start Game"
       : gameStatus === "ROUND_READY"
       ? "Next Round"
-      : gameStatus === "FINISHED"
-      ? "Play Again?"
```

Comment:
Just to check -  this is gone as it's in the popup now?

Reply:
Yeah this was moved to the winner popup